src/main/webapp/cluster-view/cluster-view.html

```diff
@@ -41,8 +41,13 @@
 
     <!-- TOP ROW #2 START -->
     <div class="row">
-        <div class="col-md-2 text-center">
-            <h4 style="color:   #009A31;">Available</h4>
+        <div class="col-md-2 text-left">
+            <div ng-show="currentCluster.availability === 'AVAILABLE'">
```

Comment:
Instead of duplicating html here, we can do it a little bit better: remove the ng-show, and directly modify properties according to availability. So instead of `class="fa fa-1x unavailable"` you would use `class="fa fa-1x" ng-class="{available: currentCluster.availability === 'AVAILABLE', unavailable: currentCluster.availability === 'UNAVAILABLE'}"`, and {{currentCluster.availability}} as paragraph content, instead of hardcoded value.